### DISCUSSION TITLE
Which is the final finn project in the result folder?

### DISCUSSION THREAD

**Discussion by MFJI** [Xilinx/finn#981] [Q&A]
Hello,
 Anybody know which is the final vivado or vitis project for finn that complete the synthesis, implementation and generate bitfile?
Thank you.

**Comment by Sekijoju** [comment]
I think you are looking for the one starting with "vivado_zynq_proj_"

**Reply by MFJI** [reply]
Thank you for your answer. I am using a vitis process so I don't think I have "vivado_zynq_proj_". Do you have an idea which is the one in a vitis process?
